ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModule.java

```diff
@@ -539,6 +551,37 @@ public void sendAccessibilityEvent(int tag, int eventType) {
     mUIImplementation.sendAccessibilityEvent(tag, eventType);
   }
 
+  public void emitUpdateDimensionsEvent() {
```

Comment:
I fixed this now. One interesting detail is that the dimensions part of constants is now a `WritableMap` instead of a `HashMap`. I wasn't sure what the impact of this would be. It built and ran fine.